index.js

```diff
@@ -1,27 +1,32 @@
 // This will check if the node version you are running is the required
 // Node version, if it isn't it will throw the following error to inform
 // you.
-if (Number(process.version.slice(1).split(".")[0]) < 8) throw new Error("Node 8.0.0 or higher is required. Update Node on your system.");
+if (Number(process.version.slice(1).split(".")[0]) < 12) throw new Error("Node 12.0.0 or higher is required. Update Node on your system.");
 
 // Load up the discord.js library
 const Discord = require("discord.js");
 // We also load the rest of the things we need in this file:
 const { promisify } = require("util");
 const readdir = promisify(require("fs").readdir);
 const Enmap = require("enmap");
+const config = require("./config.js");
 
 // This is your client. Some people call it `bot`, some people call it `self`,
 // some might call it `cootchie`. Either way, when you see `client.something`,
-// or `bot.something`, this is what we're refering to. Your client.
-const client = new Discord.Client();
+// or `bot.something`, this is what we're referring to. Your client.
+const client = new Discord.Client({
+  ws: {
+    intents: config.intents
+  }
+});
 
 // Here we load the config file that contains our token and our prefix values.
-client.config = require("./config.js");
+client.config = config
 // client.config.token contains the bot's token
 // client.config.prefix contains the message prefix
 
 // Require our logger
-client.logger = require("./util/logger.js");
```

Comment:
Whilst I know this is perfectly valid, the change you made goes against pre-established notions (Line 12 for example), please revert this change.

Reply:
ok